examples/injection/proto/injection.proto

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+import "oak/proto/handle.proto";
+
+package oak.examples.injection;
+
+message BlobStoreInterface {
```

Comment:
Isn't this the same (or very similar) as what we usually call "invocation" in the code?

Reply:
I think it is similar yes. But in this case the handles can be used to serve multiple requests, so I'm not sure we should call it an 'invocation'.